en/docs/modules/misc/MagneticConnector.md

```diff
@@ -2,17 +2,36 @@
 
 <img src="https://docs.retrogadgets.game/api/modules/MagneticConnector.png" width="200" align="right">
 
-Magnetic connects allow separated boards to connect and disconnect at set points. Unfortunately, magnetic connectors are essentially just magnets and do not report what they're connected to. Boards connected to each other with a powered magnet (light green, toggleable with the button) will be picked up as one.
-
-ℹ️ By knowing the state of all connectors, it is possible in certain configurations to know the arrangement of connectable boards.
+Magnetic connects allow separated boards to connect and disconnect at set points. Boards connected to each other with a powered magnet (light green, toggleable with the button) will be picked up as one.
 
 ## Properties
 
+### AttachedConnector - `MagneticConnector` **[Read only]**
+The other `MagneticConnector` connected to this one.
+
 ### ButtonState - `boolean` **[Read only]**
 The state of the physical button on the connector. `true` if the button is being held.
 
 ### IsConnected - `boolean` **[Read only]**
-`true` if the connector is connected to another one. The only indication of connectivity. 
+`true` if the connector is connected to another one. The only indication of connectivity.
+
+### Motherboard - `Motherboard` **[Read only]**
+The `Motherboard` attached with this `MagneticConnector`.
+
+### Type - `string` **[Read only]**
+Always return `MagneticConnectorEvent`.
+
+## Event - `MagneticConnectorEvent`
+### For Retro Gadgets V0.1.5+
+The event emitted as part of the [CPU](./CPU.md) event system.
+
+Sent when a `MagneticConnector` is connected to another.
+
+### IsConnected - `boolean` **[Read only]**
+`true` if the connector is connected to another one. The only indication of connectivity.
+
+### Type - `string` **[Read only]**
+Always return `MagneticConnectorEvent`.
 
 ## Remarks
-This should hopefully report more information in the full release. For now, have a limited set of checkable configurations, or keep track of all connectors and detect which two switch when boards are connected or disconnected, in order to track what is connected to what.
\ No newline at end of file
+This should hopefully report more information in the full release.
```

Comment:
You can probably remove this whole section now. This might be all the information we get, and it's probably good enough.

--------------------------------------------------------------------------
<small>ℹ️ Unable to generate suggested change: Suggested changes are not supported on removed lines.</small>